plugins/mathjax/plugin.js

```diff
@@ -306,55 +307,85 @@
 				if ( CKEDITOR.env.ie )
 					iFrame.removeAttribute( 'src' );
 
-				doc.write( '<!DOCTYPE html>' +
-							'<html>' +
-							'<head>' +
-								'<meta charset="utf-8">' +
-								'<script type="text/x-mathjax-config">' +
-
-									// MathJax configuration, disable messages.
-									'MathJax.Hub.Config( {' +
-										'showMathMenu: false,' +
-										'messageStyle: "none"' +
-									'} );' +
-
-									// Get main CKEDITOR form parent.
-									'function getCKE() {' +
-										'if ( typeof window.parent.CKEDITOR == \'object\' ) {' +
-											'return window.parent.CKEDITOR;' +
-										'} else {' +
-											'return window.parent.parent.CKEDITOR;' +
-										'}' +
-									'}' +
-
-									// Run MathJax.Hub with its actual parser and call callback function after that.
-									// Because MathJax.Hub is asynchronous create MathJax.Hub.Queue to wait with callback.
-									'function update() {' +
-										'MathJax.Hub.Queue(' +
-											'[ \'Typeset\', MathJax.Hub, this.buffer ],' +
-											'function() {' +
-												'getCKE().tools.callFunction( ' + updateDoneHandler + ' );' +
-											'}' +
-										');' +
-									'}' +
-
-									// Run MathJax for the first time, when the script is loaded.
-									// Callback function will be called then it's done.
-									'MathJax.Hub.Queue( function() {' +
-										'getCKE().tools.callFunction(' + loadedHandler + ');' +
-									'} );' +
-								'</script>' +
-
-								// Load MathJax lib.
-								'<script src="' + ( editor.config.mathJaxLib ) + '"></script>' +
-							'</head>' +
-							'<body style="padding:0;margin:0;background:transparent;overflow:hidden">' +
-								'<span id="preview"></span>' +
-
-								// Render everything here and after that copy it to the preview.
-								'<span id="buffer" style="display:none"></span>' +
-							'</body>' +
-							'</html>' );
+				doc.write(
+					`<!doctype html>
```

Comment:
Unfortunately, template strings can't be used as CKEditor 4 needs to run also in older browsers, especially Internet Explorer 11. Could you change it to the version with string concatenation?

Reply:
got it will look into it